### ISSUE TITLE
[CTF/20v20] Tuscany Hills

### ISSUE THREAD

**Issue by TjebbeLeen** [OvercastCommunity/CommunityMaps#525]
### Checklist
Check what applies to you. *Add an X between the brackets or click the checkboxes when you have submitted the issue.*
- [x] I have [pruned](https://pgm.dev/docs/guides/packaging/pruning-chunks) the map
- [x] I have agreed with assigning the CC BY-SA license to this map, as mentioned in the README
- [x] I have created an XML file
- [x] I have created a map image
- [x] I have uploaded the map zip file to a file sharing service
- [x] The map has been tested locally to make sure it works

# Map Name
Tuscany Hills

## Gamemode & Map Description  
Tuscany Hills is a Capture The Flag map. Teams can be up to 16 big in playersize. The objective of the game is to capture the enemy their flag 3 times in total. You can cap at your spawn or at your flag. Flag holders lose their weapons upon stealing. Players receive a gapple upon killing a player. There is a time limit of 10 minutes. When there is a tie there are 2 minutes of overtime, first team to capture the flag wins. Otherwise its a tie

## Team Sizes
2 teams, red and blue. 16 players max per team (16 v 16)

## Screenshots
https://imgur.com/a/g4lLR7c

![2022-10-14_13 58 07](https://user-images.githubusercontent.com/85294819/195849395-0ce4ea48-5b55-4ef2-8084-a2d0703ebfca.png)

# XML
https://gist.github.com/zzuf-fer/271a7b097597624f73fd7d12dd3c0cbe

# Map Image
![map](https://user-images.githubusercontent.com/85294819/195840133-ab667db1-9f4c-4479-8180-4e3bf59ee835.png)

# Download Link
https://www.mediafire.com/file/uooh8lsyvwqt40c/TuscanyHillsV2.zip/file

**Comment by zzuf-fer** [comment]
**1.0.1 (XML):** Players who intentionally drop the flag no longer get Regeneration - an effect intended for newly-spawned players and players that have just captured a flag.

**Comment by Cazadorian** [comment]
> **1.0.1 (XML):** Players who intentionally drop the flag no longer get Regeneration - an effect intended for newly-spawned players and players that have just captured a flag.

Updated in cd08643, further update loaded in 73abe57

**Comment by zzuf-fer** [comment]
**1.0.3 (Map):** Zynnphony has added beacons to the flag respawn points.
_New download link:_ https://mega.nz/file/rvBnlAKa#vT0mbwDHp0afV_96CtSAmqDGYhmokL984UMLkKofHLY
**1.0.3 (XML):** Simplified net regions, removed redundant region.

**Comment by zzuf-fer** [comment]
**Tuscany Hills 1.0.4**: XML Update
- Team sizes increased from 16 to 20.
- New tip, and several messages have been modified.
- Flag recover time has been increased from 5 seconds to 10 seconds.
- Flag carriers are now given Sticks with Knockback I, and Speed I.

**Comment by Cazadorian** [comment]
> **Tuscany Hills 1.0.4**: XML Update
> 
> * Team sizes increased from 16 to 20.
> * New tip, and several messages have been modified.
> * Flag recover time has been increased from 5 seconds to 10 seconds.
> * Flag carriers are now given Sticks with Knockback I, and Speed I.

Updated in 220c71b

**Comment by TjebbeLeen** [comment]
I see your point with removing power 1. The main issue I have with most gamemodes right now is that it is incredibly difficult to recover your wool (or flag in this case) from a distance. Most gamemodes you need to hit at least 7 but more likely 9-10 arrows on a player consecutively before you kill them. This is near impossible. So I'd like to suggest adding projectile protection to the chain leggings, and removing the projectile protection from the player when they pick up the flag. This keeps sword damage the same, whilst making it slightly easier to recover the flag with bow.

And you're right, the swiftness is too much. I personally hate the slowness effect most flag maps apply when you pick up the flasg. Keeping the speed the same is the best middle ground imo. Let the players keep their weapons too instead of the knockback stick. OCC is a different server and this would probably the best way forwards.

tl:dr
Add projectile prot to chain leggings
Remove projectile prot from chain leggings when player picks up the flag
Let players keep the normal weapon kit when picking up the flasg (no knockback stick)

@Cazadorian

**Comment by Cazadorian** [comment]
This map has been in the pool for a few months and after the most recent update it seems to play decently within the pool. As a result I'm closing this issue as complete for now, thanks for the submission.
